examples/xmetrics.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: metrics.crossplane.io/v1
+kind: ClusterMetric
+metadata:
+  name: clustermetric-sample
+spec:
+  matchName: ".crossplane.io"
+---
+apiVersion: metrics.crossplane.io/v1
+kind: Metric
+metadata:
+  name: metric-sample
+  namespace: upbound-system
+spec:
+    matchName: "metrics.metrics.crossplane.io"
+---
+apiVersion: metrics.crossplane.io/v1
+kind: ClusterMetric
+metadata:
+  name: upbound-metric-sample
+spec:
+  matchName: ".upbound.io"
```

Comment:
This creates a lot of custom metrics, can you provide an example with a list of common families instead?